src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,30 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards) to manage releases. Read
+“[Tracking progress on your project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards)"
+to learn more on how to work with projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board](https://github.com/orgs/submariner-io/projects/15) hosts issues or features that are not scheduled
+for the release currently being developed.
```

Comment:
@mkolesnik where do you think is best place to track bugs?
Since a project board is giving aggregated view from all our repositories, it make sense to have the bugs tracked on either of our project boards (backlog or current version)

Reply:
I think we should track any issue open in backlog board.

Reply:
We can track bugs with queries (as we do today), I don't see a particular reason to add them to backlog but if it makes sense to everyone else I won't object.